lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -554,6 +554,18 @@ defmodule ExUnit.Assertions do
       {name, meta, context}, acc when is_atom(name) and is_atom(context) ->
         {:ok, [{name, [generated: true] ++ meta, context} | acc]}
 
+      node = {:<<>>, meta, context}, acc when is_list(context) ->
```

Comment:
We cannot assume all sigils will define variables. Otherwise this will break code such as this `assert ~s(a) = "a"`.